<!-- -*- mode: XML -*- -->
<launch>
    <arg name="qb_doking_dxl_id"            default="1"/>
    <arg name="qb_doking_dxl_serial"        default="/dev/ttyUSB0"/>
    <arg name="qb_doking_dxl_baud"          default="57600"/>
    <arg name="qb_doking_dxl_pwm_treshold"  default="800"/>
    <arg name="qb_doking_dxl_load_treshold" default="520"/>
    <arg name="qb_doking_verbose"           default="False"/>
    <arg name="output"                      default="screen"/>

    <node pkg="qb_docking_station" type="qb_docking_station_node.py" name="qb_docking_station" ns="qb_docking_station_mngr" 
            output="$(arg output)">
        <param name="qb_doking_dxl_id"              value="$(arg qb_doking_dxl_id)"/>
        <param name="qb_doking_dxl_serial"          value="$(arg qb_doking_dxl_serial)"/>
        <param name="qb_doking_dxl_baud"            value="$(arg qb_doking_dxl_baud)"/>
        <param name="qb_doking_dxl_pwm_treshold"    value="$(arg qb_doking_dxl_pwm_treshold)"/>
        <param name="qb_doking_dxl_load_treshold"   value="$(arg qb_doking_dxl_load_treshold)"/>
        <param name="qb_doking_verbose"             value="$(arg qb_doking_verbose)"/>
    </node>

</launch>